audit/audit.go

```diff
@@ -43,6 +48,10 @@ type BackendConfig struct {
 
 	// Config is the opaque user configuration provided when mounting
 	Config map[string]string
+
+	// auditing to stdout (file audit backend) requires the logger
+	// to prevent interleaving w/ the main log output
+	Logger log.Logger
```

Comment:
Logging goes to stderr after startup, not stdout, so this isn't necessary.

Reply:
@jefferai I'm confused then by your comments on #2195. What were you referring to when you said "I worry about interleaving issues here given that Vault also writes things to stdout at various points."? I guess I wrongly assumed you were concerned about interleaving between a stdout audit backend and vault's main logging.

Reply:
When Vault starts it dumps a bunch of output to stdout. Then the server logs go to stderr. However, making stdout a separate audit backend doesn't make any difference. You'd basically just need to ignore any line that isn't JSON.